app/models/setting.rb

```diff
@@ -80,6 +80,11 @@ def self.stick_general_first
     (where(:category => sticky_setting) + where.not(:category => sticky_setting)).group_by(&:category)
   end
 
+  def self.replace_keyword(keyword)
+    keyword.gsub! '$VERSION', _("Version") + " " + SETTINGS[:version].version
+    keyword
+  end
```

Comment:
It is a not good practice to modify arguments, just use `gsub` which will return a new string.

Reply:
Also this method name could be called `replace_keywords` as we can add more later.